src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -150,6 +153,27 @@ class ContentSidebar extends PureComponent<Props, State> {
         });
     }
 
+    /**
+     * Gets the user avatar URL
+     *
+     * @param {string} id the user id
+     * @return the user avatar URL string for a given user with access token attached
+     */
+    async getAvatarUrl(id: string): Promise<?string> {
```

Comment:
Should I move this to Users API?

Reply:
Sure...

Reply:
I guess then it becomes similar to File.getDownloadUrl()